.master-plan.yml

```diff
@@ -196,9 +196,17 @@ spec:
     description: "Give IO Stress on the Kubernetes Node"
     labels:
       test/feature: "Generic-Infra"
-      test/tags: "node, io,disk,stress"
+      test/tags: "node, io, disk, stress"
       git/location: "https://github.com/litmuschaos/litmus-e2e/tree/gh-pages/generic-pipeline/node-io-stress"
-      test/status: "Done"      
+      test/status: "Done"
+
+  - tcid: TCID-EC2-GENERIC-INFRA-NODE-SELECTOR
+    name: "TCID-EC2-GENERIC-INFRA-NODE-SELECTOR"
+    description: "Node selector functionality for given target node or node label"
+    labels:
+      test/feature: "Generic-Infra"
+      test/tags: "node, selector"
+      git/location: "https://github.com/litmuschaos/litmus-e2e/tree/gh-pages/generic-pipeline/node-selector"
```

Comment:
So we'll be adding this in upcoming PR right?

Reply:
I think I didn't understand the requirement properly, can you please explain me what exactly I need to do?

Reply:
we use to add some details about the tests in `gh-pages` branch! like for [pod-delete-test](https://github.com/litmuschaos/litmus-e2e/tree/gh-pages/generic-pipeline/pod-delete) but this needs to be done in another PR as it is on diff branch.

Reply:
Sure, will make a seperate PR for it shortly.